src/Symfony/Component/Validator/ConstraintValidator.php

```diff
@@ -105,6 +111,37 @@ protected function formatValue($value, $format = 0)
             return $value->format('Y-m-d H:i:s');
         }
 
+        if (($format & self::PRETTY_DATE_INTERVAL) && $value instanceof \DateInterval) {
+            $formattedValueParts = [];
+            foreach ([
+                'y' => 'year',
+                'm' => 'month',
+                'd' => 'day',
+                'h' => 'hour',
+                'i' => 'minute',
+                's' => 'second',
+                'f' => 'microsecond',
```

Comment:
this formatting logic is English-only, which looks bad.

Reply:
As we did for dates just above, the format is hardcoded here except in this case there are words.